composer.json

```diff
@@ -4,7 +4,7 @@
     "keywords": ["cqrs", "event sourcing", "domain-driven design", "ddd"],
     "license": "MIT",
     "require": {
-        "beberlei/assert": "~2.0",
+        "beberlei/assert": "~3.0",
```

Comment:
Why not allow both `~2.0 || ~3.0`. There is no real reason to only required 3.0.

Reply:
There is not a hard reason, but also doesn't add a lot of value imo. 
We require PHP7, beberlei/assert 3.0 is released for PHP7, so I see no reason not make that the requirement.